### PR TITLE
2780: upload winter assets

### PR DESCRIPTION
**Pull request by ltello**
### Context
[Card](https://trello.com/c/Ipuh1V9v)

### Changes proposed in this pull request
 Update the job to upload assets to the new csv structure received from CC

### CODE REVIEW

File: app/jobs/computacenter_asset_job.rb [hunk 1]
```diff
@@ -70,21 +71,35 @@ def line_count_of_file(path)
...
   end
 
   def import_csv_row(row)
     Asset.create!(attributes_hash(row))
   end
 
+  # def attributes_hash(row)
+  #   # we don't store row[0] (`sys_id`) nor row[14] (`sys_updated_at`)
+  #   { tag: row[1], serial_number: row[2], model: row[3], department: row[4], department_id: row[5], department_sold_to_id: row[6], location: row[7], location_id: row[8], location_cc_ship_to_account: row[9], bios_password: row[10], admin_password: row[11], hardware_hash: row[12], sys_created_at: row[13] }
+  # end
```
Comment (markhallen) [inline app/jobs/computacenter_asset_job.rb:85]: Do we need this?

File: app/jobs/computacenter_asset_job.rb [hunk 2]
```diff
@@ -70,21 +71,35 @@ def line_count_of_file(path)
...
+    {
+      tag: row['OrderNumber'],
+      serial_number: row['SerialNumber'],
+      model: row['MaterialDescription'],
+      department: row['SoldToCustomer'],
+      department_id: row['URN'],
+      department_sold_to_id: row['SoldToAccountNo'],
+      location: row['ShipToCustomer'],
+      location_id: row['SchoolURN'],
+      location_cc_ship_to_account: row['ShipToAccountNo'],
```
Comment (markhallen) [inline app/jobs/computacenter_asset_job.rb:97]: Better! row[1] etc was confusing.

**Review by markhallen — APPROVED** [review]
Nice touch adding in the new syntax 😁

File: app/jobs/computacenter_asset_job.rb (hunk 1 above)
Comment (ltello) [reply app/jobs/computacenter_asset_job.rb]: Not actually. Removed.

**Comment by ltello** [comment]
Thanks for the approval!

**Comment by ltello** [comment]
[Explore including wave 8+ serial numbers in asset search tool](https://trello.com/c/Ipuh1V9v/2780-explore-including-wave-8-serial-numbers-in-asset-search-tool)

**Pull request merged**